projects/bin-picking-with-vision/application.py

```diff
@@ -0,0 +1,210 @@
+import math
+import numpy as np
+import cv2
+from PIL import Image
+
+from jacobi import Obstacle, Motion, Frame, Planner, Studio, PointCloud, CartesianWaypoint
+from jacobi_vision.drivers import SimulatedCameraDriver
+from jacobi_vision import ImageType, ColorImage
+
+
+class VisionBinPicking:
+    def __init__(self, project_path: str):
```

Comment:
```suggestion
    def __init__(self, project_path: Path):
```
but need to import `from pathlib import Path` for that.